### DISCUSSION TITLE
想问一下给X6官网贡献demo的流程和标准是什么

### DISCUSSION THREAD

**Discussion by cjinhuo** [antvis/X6#1817] [Q&A]
调研X6时，写了个demo包含的功能有（在官网案例中没有找到类似的demo）：
1. 拖拽新增节点、从节点边上的链接桩拖拽出新的边到节点
2. 每个节点（分组是特殊的节点）有上下左右四个连接桩
3. 两个节点之间只能允许用户手动连一条边
4. 节点、边label使用react组件渲染
5. 鼠标左键节点、边触发侧边栏，右键节点、边触发菜单
6. 键盘快捷键、鼠标右键可删除节点、边、分组
7. 自定义节点大小：节点、折叠状态下的分组可自定义大小（展开状态下的分组需要根据组内的节点计算大小和位置）
8. 画布的滚动、小地图、放大、缩小
9. 节点、边的操作undo、redo功能
10. 节点、边上的指标实时更新
11. 支持查看、编辑模式的切换

分组功能：
1. 框选多个，并可新建多层分组（一个节点只允许在一个分组中）
2. 可以拖拽单个节点嵌入分组，当可以嵌入时会高亮该分组边框
3. 拖拽分组内的点时，实时计算外层分组的大小以及位置
4. 折叠分组时自动生成边以及边的聚合关系，当有分组实体线的时，自动生成线则会被替换

**Comment by NewByVector** [comment]
🥳🥳 可以先提 PR 到 https://github.com/antvis/X6/tree/master/examples/x6-example-features 目录下面。

**Reply by xiazhaohui** [reply]
没找到诶

**Comment by zrx1118** [comment]
> 
请问demo在哪里可以看，我主要想看一下边聚合。谢谢
